k8s.gcr.io/images/k8s-staging-bootkube/OWNERS

```diff
@@ -0,0 +1,6 @@
+# See the OWNERS file documentation:
+#  https://github.com/kubernetes/kubernetes/blob/master/docs/devel/owners.md
```

Comment:
```suggestion
# See the OWNERS docs at https://go.k8s.io/owners
```

The link is broken. See my suggestion (which is what we use in most of the OWNERS files).